websites/P/PresenceDB/presence.ts

```diff
@@ -0,0 +1,103 @@
+const presence = new Presence({
+  clientId: "840759396103749633"
+});
+
+async function getTranslation(name: string) {
+  const language = (await presence.getSetting("lang")) === 0 ? "en" : "de";
+
+  switch (name) {
+    case "home":
+      switch (language) {
+        case "de":
+          return "Durchsucht die Startseite";
+        default:
+          return "Browsing home page";
+      }
+    case "search":
+      switch (language) {
+        case "de":
+          return "Sucht nach etwas";
+        default:
+          return "Searching for something";
+      }
+    case "faq":
+      switch (language) {
+        case "de":
+          return "Durchsucht das FAQ";
+        default:
+          return "Browsing FAQ page";
+      }
+    case "activity":
+      switch (language) {
+        case "de":
+          return "Schaut sich Aktivität an";
+        default:
+          return "Viewing activity";
+      }
+    case "user":
+      switch (language) {
+        case "de":
+          return "Schaut sich Benutzer an";
+        default:
+          return "Viewing user";
+      }
+    case "settings":
+      switch (language) {
+        case "de":
+          return "Durchsucht die Einstellungen";
+        default:
+          return "Browsing settings page";
+      }
+    case "viewUser":
+      switch (language) {
+        case "de":
+          return "Benutzer anzeigen";
+        default:
+          return "View user";
+      }
+    case "viewActivity":
+      switch (language) {
+        case "de":
+          return "Aktivität anzeigen";
+        default:
+          return "View activity";
+      }
+  }
+}
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp: new Date().getTime()
```

Comment:
Since it's in a loop here so... the time will always be the current time, and you need to `/ 1000` it

Reply:
Makes sense, gonna change that later.

Reply:
But what's really weird is that is actually worked that way...

Reply:
it works because the app doesn't update the presence if the timestamp only changed 1ms
But these are still unneeded updates to the app.